easybuild/easyconfigs/m/MEGAHIT/MEGAHIT-1.1.2-foss-2018a.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'MakeCp'
+
+name = 'MEGAHIT'
+version = '1.1.2'
+
+homepage = 'https://github.com/voutcn/megahit'
+description = """An ultra-fast single-node solution for large and complex
+ metagenomics assembly via succinct de Bruijn graph"""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['https://github.com/voutcn/%(namelower)s/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['d0d3965dd49c6fdaea958ef66146cb6b30b7d51acbbfe94194c437f15a424cb5']
+
+# This version is CPU-only.  It's also possible to compile this with
+# CUDA, but that isn't working yet (and requires libcuda.so!).
```

Comment:
@tutufan Hmm, not sure if it's worth keeping this comment in?

Not having CUDA as dependency, and the `gpu=0` make it fairly clear this is a non-GPU build?